holidayextras/ui-brand-heha
# Modal background colour variables

#### What does this PR do?
The HEHA theme is "inverted", i.e. light text on a dark background.
Modals do not automatically pull through the body background colour or text colour so we need to set the variables.

1. Background colour set to the same as the body colour (purple), allowing for the white text to be readable.
2. Dark purple borders to define the element boundaries.
3. Padding for a little more spacing.


#### What tests does this PR have?
N/A


#### How can this be tested?
First you'll need to re-compile the .less.

Then create a HEHA! page by making a carpark booking with a flight to Barcelona & go through to holiday manager. Grab the rails-id (the number at the end of the url) & add it to localhost:5000/#heha/your-rails-id.

You can see the results of this PR by viewing the modals "Login" and "Policy".


#### Screenshots / Screencast
I don't have an actual HEHA example to show you, but it should look something like this example.
Ignore the content differences, you're specifically looking for the 3 things mentioned above.

![heha-modal](https://cloud.githubusercontent.com/assets/400087/10789330/afb7cc98-7d74-11e5-9161-3bf86501dc4b.png)


#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/qgBzaK9vz3edG/giphy.gif)


#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
